src/eterna/AnnotationManager.ts

```diff
@@ -1042,15 +1020,15 @@ export default class AnnotationManager {
             const view = this.getAnnotationView(params.pose, i, params.item);
             if (params.item.type === AnnotationHierarchyType.ANNOTATION) {
                 view.onMovedAnnotation.connect((point: Point) => {
-                    const anchorIndex = this._annotations[params.itemIndex].positions[i].anchorIndex;
+                    const anchorIndex = this.allAnnotations[params.itemIndex].positions[i].anchorIndex;
```

Comment:
`item` vs `this.allAnnotations[params.itemIndex]`

Reply:
Good catch